expense-tracker/src/db/procedures/fetch_user_details_v1dot0.sql

```diff
@@ -3,6 +3,6 @@ create procedure fetch_user_details_v1dot0 (
     IN in_user_id bigint
 )
 BEGIN
-    select * from t_user_details where f_id = in_user_id;
+    select * from t_user_details where f_email = in_user_id;
```

Comment:
is unique constraint added for userId at table ? if so, please handle unique constraint exception.